Explorer/Assets/DCL/Navmap/FloatingPanelController.cs

```diff
@@ -24,6 +24,9 @@ public class FloatingPanelController : IDisposable
 
         private CancellationTokenSource cts;
 
+        private readonly Vector2 rectTransformLocalPosition = new Vector3(1702, 480);
+        private readonly Vector2 rectTransformLocalPositionOutside = new Vector3(2100, 480);

```

Comment:
It would be nice to leave a comment here about what these hardcoded numbers mean.